### ISSUE TITLE
[Sentry] ModuleIntegration wasn't able to extract modules: module integration failed

### ISSUE THREAD

**Issue by dmitryyankowski** [dgraph-io/dgraph#5166]
### What version of Dgraph are you using?
Master

### Have you tried reproducing the issue with the latest release?
Yes

### What is the hardware spec (RAM, OS)?
24GB Memory and Ubuntu 18.04.4 LTS

### Steps to reproduce the issue (command/config used to run Dgraph).
Ctrl-c on my Dgraph zero instance and received the following error.

![image](https://user-images.githubusercontent.com/20114263/79001111-eb493b80-7b1b-11ea-9208-883a8911e65a.png)

```
^CI0410 10:55:42.313708    1256 run.go:266] --- Received interrupt signal
I0410 10:55:42.314014    1256 run.go:285] Shutting down...
E0410 10:55:42.314227    1256 http.go:252] Stopped taking more http(s) requests. Err: accept tcp [:
:]:6080: use of closed network connection
panic: send on closed channel

goroutine 228 [running]:
github.com/dgraph-io/dgraph/raftwal.(*DiskStorage).CreateSnapshot(0xc0064eaf80, 0x21e79a, 0xc00044e
b58, 0xc000300840, 0x15f, 0x15f, 0x0, 0x0)
        /home/dmitry/dgraph/raftwal/storage.go:692 +0x368
github.com/dgraph-io/dgraph/dgraph/cmd/zero.(*node).trySnapshot(0xc0000f3680, 0x0)
        /home/dmitry/dgraph/dgraph/cmd/zero/raft.go:614 +0x1c9
github.com/dgraph-io/dgraph/dgraph/cmd/zero.run.func3(0xc000496ac0, 0xc00734e180, 0x1c2d000, 0xc000
0f4060, 0xc0064eaf80, 0x1c2d000, 0xc0000f4040)
        /home/dmitry/dgraph/dgraph/cmd/zero/run.go:297 +0x1b7
created by github.com/dgraph-io/dgraph/dgraph/cmd/zero.run
        /home/dmitry/dgraph/dgraph/cmd/zero/run.go:282 +0xf70
[Sentry] 2020/04/10 10:55:42 ModuleIntegration wasn't able to extract modules: module integration f
ailed
[Sentry] 2020/04/10 10:55:42 Sending fatal event [cc19eb68a1de4cdc8f4808283e9d17ea] to sentry.io pr
oject: 1805390
[Sentry] 2020/04/10 10:55:42 Buffer flushed successfully.
```

### Expected behaviour and actual result.

Sentry should not output this error? :D

**Comment by MichelDiz** [comment]
Is Sentry already released? if not so, it is normal to have issues in master. Certainly, someone is still finishing something.

Ping @parasssh

**Comment by parasssh** [comment]
Sentry Integration is on master as well as recently released v20.03.0 

I have personally never seen the [Sentry] ModuleIntegration wasn't able to extract modules: module integration failed. It seems to be an issue on the Sentry's Go SDK. I will open a bug with Sentry and follow up.

**Comment by dmitryyankowski** [comment]
Ah sorry @MichelDiz Yeah I use the master branch normally so I can catch bugs and such early on and help report them :) 

@parasssh Yeah i've only gotten this error 1 time in the numerous amount of times I have ctrl-c'd the zero process. Just wanted to make sure it wasn't an issue in the Dgraph code!

**Comment by parasssh** [comment]
Opened an Issue with Sentry.

**Comment by rhcarvalho** [comment]
Hi there! I'm one of the Sentry Go SDK maintainers.

The log message can be disregarded, it is not an error, but rather information that the SDK could not get module information to attach to the error prior to reporting. At the moment it requires access to source code / `go.mod`.

The message might be too noisy/scary. We can at least rephrase it.

**Comment by rhcarvalho** [comment]
We've released [sentry-go v0.6.0](https://github.com/getsentry/sentry-go/releases/tag/v0.6.0), among other improvements it includes a complete rewrite of the Modules integration. Now modules are read from the Go runtime for binaries built with Go Module support enabled. For other binaries the modules integration is disabled.

**Comment by dmitryyankowski** [comment]
@parasssh :)
